autoload/direnv.vim

```diff
@@ -0,0 +1,83 @@
+" direnv.vim - support for direnv <http://direnv.net>
+" Author:       zimbatm <http://zimbatm.com/> & Hauleth <lukasz@niemier.pl>
+" Version:      0.3
+
+scriptencoding utf-8
```

Comment:
No need for that, as there are no non-ASCII characters in this file.

Reply:
I’ve generally taken the habit of putting that in there all the time. It’s not necessary, but it shouldn’t hurt.